utils/userDataService.ts

```
@@ -33,6 +33,5 @@ export async function computeMetadataHash(
 
 export const getPfpFromFullId = (identity: FullId): string => {
   if (identity && identity.pp_url) return getImgUrl(identity.pp_url);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)